console/src/components/MapCore/MapCore.js

```diff
@@ -560,16 +581,55 @@ class MapCoreBase extends React.Component {
 		delete rest.proposedDestination;
 		delete rest.setDestination;
 		delete rest.skin;
+		delete rest.topLocations;
 		delete rest.updateNavigation;
 		delete rest.viewLockoutDuration;
 		delete rest.zoomToSpeedScaleFactor;
+		const {duration, distance, eta, selectedDestination, selfDriving} = this.state;
+
 		return (
 			<div {...rest} className={classnames(className, css.map)}>
 				{this.message ? <div className={css.message}>{this.message}</div> : null}
-				<nav className={css.tools}>
-					{tools}
```

Comment:
I gather that this means that MapCore no longer has support for custom buttons defined by the consumer? It would be great if this feature was restored. Just a place to inject externally provided children.

Reply:
Yes, I liked to have it, but I wasn't sure what to put where.. I will put those back, but I need to think about it more about where to extract and move the buttons to.